src/main/java/com/featureprobe/sdk/server/DatetimeMatcher.java

```diff
@@ -0,0 +1,8 @@
+package com.featureprobe.sdk.server;
+
+@FunctionalInterface
+public interface DatetimeMatcher {
+
+    boolean match(long target, long customValue);
```

Comment:
Consistent with other Matchers
```suggestion
    boolean match(long target, List<String> objects);
```